packages/integration/test/load/loadTest.ts

```diff
@@ -0,0 +1,81 @@
+import { ethers} from "ethers";
+import { Config, getConfig } from "../utils/config";
+import { setupChainIntegration, startContainers } from "../utils/containerManager";
+import { SdkAgent } from "./loadTestSdkAgent";
+
+import { LoadTestConfig, TestTargets } from "./loadTestTypes";
+import { PingPong } from "./pingPongTest";
+
+class LoadTestEnvironment{
+  private config:Config;
+  private testConfig:LoadTestConfig;
+  private containersUp!:boolean;
+
+   constructor(testParams:LoadTestConfig){
+    this.config = getConfig();
+    this.testConfig = testParams;
+    if(testParams.spawnContainters){
```

Comment:
Need to get your prettier running format on save...

Reply:
woops fixed in my IDE